jobs/JGDAS_GLOBAL_AERO_ANALYSIS_INITIALIZE

```diff
@@ -0,0 +1,101 @@
+#! /usr/bin/env bash
+
+export STRICT="NO"
+source "${HOMEgfs}/ush/preamble.sh"
+
+##############################################
+# Make temp directory
+##############################################
+export DATA=${DATA:-${DATAROOT}/${RUN}aeroanl_${cyc}}
```

Comment:
Do you think, it is good to check if `DATA` exists and if it does, remove it.  Only in the initialize job.

Reply:
yeah, I committed the suggestion.